Be more general in looking for syscall.h

Signed-off-by: Douglas Jerome <djerome@crosslinux.org>

diff -Naur crosstool-ng-1.18.0-old/scripts/build/libc/glibc-eglibc.sh-common crosstool-ng-1.18.0-new/scripts/build/libc/glibc-eglibc.sh-common
--- crosstool-ng-1.18.0-old/scripts/build/libc/glibc-eglibc.sh-common	2013-01-31 13:07:53.000000000 -0700
+++ crosstool-ng-1.18.0-new/scripts/build/libc/glibc-eglibc.sh-common	2013-08-10 14:31:03.358114612 -0700
@@ -404,8 +404,22 @@
             # but it breaks arm, see http://sourceware.org/ml/crossgcc/2006-01/msg00091.html
             case "${CT_ARCH}" in
                 arm)    ;;
-                *)  CT_DoExecLog ALL cp -v "misc/syscall-list.h"            \
+                *)  if [ -f misc/syscall.h ]; then 
+                       CT_DoExecLog ALL cp -v "misc/syscall.h" \
                                            "${CT_HEADERS_DIR}/bits/syscall.h"
+                    fi;
+                    if [ -f misc/bits/syscall.h ]; then 
+                       CT_DoExecLog ALL cp -v "misc/bits/syscall.h" \
+                                           "${CT_HEADERS_DIR}/bits/syscall.h"
+                    fi;
+                    if [ -f misc/syscall-list.h ]; then 
+                       CT_DoExecLog ALL cp -v "misc/syscall-list.h" \
+                                           "${CT_HEADERS_DIR}/bits/syscall.h"
+                    fi;
+                    if [ -f misc/bits/syscall-list.h ]; then 
+                       CT_DoExecLog ALL cp -v "misc/bits/syscall-list.h" \
+                                           "${CT_HEADERS_DIR}/bits/syscall.h"
+                    fi
                     ;;
             esac
         fi
